index.html

```diff
@@ -12,13 +12,229 @@
         <!-- Place favicon.ico in the root directory -->
 
         <link rel="stylesheet" href="assets/css/styles.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.8.1/slick.min.css">
```

Comment:
If you stick the slick styles before your main css file, it becomes easier to override the slick styles.